### ISSUE TITLE
error in getRDS

### ISSUE THREAD

**Issue by CherineLee** [edward130603/BayesSpace#64]
Hi
I'm following the vignettes provided with the package and I met an error in getRDS. The command and error were as below
``` 
> melanoma <- getRDS(dataset="2018_thrane_melanoma", sample="ST_mel1_rep2")
Error: Dataset/sample not available
```

Is the dataset no longer available? If so, could you suggest other available datasets for testing?

Thank you for your time!

**Comment by edward130603** [comment]
Thanks for trying our tool. Can you see if the fix in #51 solves the problem? The issue occurs on older versions of R/Bioconductor.

**Comment by CherineLee** [comment]
Thank you for the quick reply!!!
